spec.html

```diff
@@ -45963,7 +45963,7 @@ <h1>%AsyncIteratorPrototype% [ @@asyncIterator ] ( )</h1>
 
     <emu-clause id="sec-async-from-sync-iterator-objects">
       <h1>Async-from-Sync Iterator Objects</h1>
-      <p>An Async-from-Sync Iterator object is an async iterator that adapts a specific synchronous iterator. There is not a named constructor for Async-from-Sync Iterator objects. Instead, Async-from-Sync iterator objects are created by the CreateAsyncFromSyncIterator abstract operation as needed.</p>
+      <p>An Async-from-Sync Iterator object is an async iterator that adapts a specific synchronous iterator. Async-from-Sync Iterator objects are never directly accessible to ECMAScript code. There is not a named constructor for Async-from-Sync Iterator objects. Instead, Async-from-Sync iterator objects are created by the CreateAsyncFromSyncIterator abstract operation as needed.</p>
```

Comment:
Is this just stating that ECMA-262 doesn't give a way to access Async-from-Sync iterator objects, or is it imposing a requirements on hosts to also not expose them?